Src/BootCamp.Chapter/Conversions.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+namespace BootCamp.Chapter
+{
+    class Conversions
+    {
+        public static void Demo()
+        {
+            Console.WriteLine("Convert Decimal To Binary");
+            ConvertDecimalToBinary();
+            Console.WriteLine();
+            Console.WriteLine("Convert Binary To Decimal");
+            ConvertBinaryToDecimal();
+        }
+
+        private static void ConvertDecimalToBinary()
+        {
+            int i;
+            int[] binaryHolder = new int[40];
+            int[] a = new int[40];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'a'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5011723)